File: Day-3/Q9.java
import java.util.Scanner;
import java.lang.Math;

public class Q9{
    public static void main(String[] args){
        try(Scanner s = new Scanner(System.in)){
            System.out.print("Enter a Number: ");
            int n = s.nextInt();
            int i;
            for(i = 2; i <= Math.sqrt(n); i++)
                if(n % i == 0)
                    break;
            if(i > Math.sqrt(n))
                System.out.println("Given No. is a Prime No.");
            else
                System.out.println("Given No. is not a Prime No.");
        }
    }
}

File: Day-3/Q9.cpp
#include <iostream>
#include <cmath>
using namespace std;

int prime(int n){
    int i;
    for(i = 2; i <= sqrt(n); i++)
        if(n % i == 0)
            break;
    if(i > sqrt(n))
        return n;
    return 0;
}

int main() {
    int n;
    cout << "Enter a Number : ";
    cin >> n;
    if(prime(n) != 0)
        cout << "Given No. is a Prime Number.";
    else
        cout << "Given No. is not a Prime Number.";
    return 0;
}
